pkg/system/names.go

```diff
@@ -16,17 +16,33 @@ limitations under the License.
 
 package system
 
-import "os"
+import (
+	"fmt"
+	"os"
+)
 
 var (
-	// Namespace holds the K8s namespace where our build system
-	// components run.
-	Namespace string
+	NamespaceEnvKey = "SYSTEM_NAMESPACE"
 )
 
-func init() {
-	Namespace = os.Getenv("SYSTEM_NAMESPACE")
-	if Namespace == "" {
-		Namespace = "knative-build"
+// Namespace holds the K8s namespace where our build system
+// components run.
+func Namespace() string {
```

Comment:
Can we hoist this package into `knative/pkg` first, and apply the rest of this update when picking that up?  You are going to have to touch all of these files for both anyways.

Reply:
https://github.com/knative/pkg/pull/235. Is this right?